src/dut.rs

```diff
@@ -307,13 +308,24 @@ impl DutInfo {
             }
         }
         if keys.contains(&"dut_id") {
-            let model = values.get("model");
-            let serial = values.get("serial");
-            if let (Some(Ok(model)), Some(Ok(serial))) = (model, serial) {
+            let serial: String = if let Some(Ok(serial)) = values.get("serial") {
```

Comment:
nit: do we need an explicit type (`String`) here?

Seems like it's obvious for the compiler because of the `to_string()`.

Reply:
Thanks, removed.